content/en/blog/2023/announcing-notation-rc2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Notation v1.0.0-RC.2 is available!
+author:  "Notation Release Team"
+date: 2023-02-27
+draft: false
+---
+
+The Notary v2 project maintainers are pleased to announce the release of Notation v1.0.0-RC.2, including Notation CLI, Notation-go, and Notation-core-go library. This is the first Notation release of 2023. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release adds enhancements including inspecting signatures associated with signed artifacts, storing signatures in the registry using OCI image manifest, adding user-defined metadata to signature payload, and more!  
```

Comment:
Would be nice to list all significant changes as a bulleted list.

Reply:
Good suggestion! I changed it to a bulleted list